pkg/artifactory/resource_artifactory_permission_targets_v1.go

```diff
@@ -128,6 +133,10 @@ func resourcePermissionTargetV1Delete(d *schema.ResourceData, m interface{}) err
 	permissionTarget := unpackPermissionTargetV1(d)
 	_, resp, err := c.V1.Security.DeletePermissionTargets(context.Background(), *permissionTarget.Name)
 
+	if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.